### ISSUE TITLE
Add storyshots for automated accessibility testing

### ISSUE THREAD

**Issue by stephiescastle** [nasa-jpl/explorer-1#143]
### Description

Our implementation of Storybook in WWW includes [Storyshots](https://www.npmjs.com/package/@storybook/addon-storyshots), which allows us to automate accessibility testing locally and as part of our CI/CD workflow. While automated testing doesn't catch everything, it has been useful on WWW to catch any outliers and regressions.

All of the components ported from WWW to Explorer 1 have been tested via Storyshots, but are no longer being tested as Explorer 1 does not have Storyshots set up. As Explorer 1 becomes our standard for the design system, I think we should add automated accessibility testing to our workflow here as well.

### Task list

- [ ] Add storyshots configuration for html stories using [axe-core](https://github.com/dequelabs/axe-core) (check config in www)
- [ ] Add storyshots to our ci/cd checks

**Comment by stephiescastle** [comment]
Completed in #484. Not using storyshots, but using Storybook's native test-runner.
